build-scripts/build_llvm.py

```diff
@@ -45,6 +45,7 @@ def build_llvm(llvm_dir, platform, backends, projects):
         "-DLLVM_ENABLE_IDE:BOOL=OFF",
         "-DLLVM_ENABLE_TERMINFO:BOOL=OFF",
         "-DLLVM_ENABLE_ZLIB:BOOL=OFF",
+        "-DLLVM_ENABLE_LIBEDIT:BOOL=OFF",
```

Comment:
Had better not change this file currently, the line is to be added in https://github.com/bytecodealliance/wasm-micro-runtime/pull/1878

Reply:
oops, this was not intentional. i will revert.

Reply:
reverted